src/core/syscalls/syscall_info.rs

```diff
@@ -1,4 +1,12 @@
 pub fn get_syscall_size_from_name(syscall_name: &str) -> usize {
+    match syscall_name {
+        "emit_event" => 4,
+        "deploy" => 9,
+        _ => unreachable!(),
```

Comment:
Are we sure this is not reached?

Reply:
you are right, this isn't unreachable. This is just unimplemented. I will change that.